matcher/service/routes.py

```diff
@@ -97,6 +107,8 @@ def match():
 
         interface.set_status(MatcherStatus.QUEUED)
 
+        interface.validate_group(interface.match_group)
```

Comment:
moving the call here is not going to reduce the response time in the POST /match call, why you don't want to put inside the task?